e2e/cypress/integration/messaging/emoji_size_spec.js

```diff
@@ -61,4 +73,20 @@ describe('Messaging', () => {
         // * Verify emoji size
         checkEmojiSize('@spacesMessage', emojis);
     });
+
+    it('MM-15012 - Emojis are not jumbo when accompanied by text', () => {
+        const emojis = [':book:', ':key:', ':gem:'];
+
+        // # Post a message
+        const messageText = 'This is a message from the future';
+        cy.postMessage(messageText + ' ' + emojis);
```

Comment:
I'm fine with this instead of ``cy.postMessage(messageText + ' ' + emojis.join(' '));`` so we have another catch whether emoji will render if followed by a comma (`,`).
![Screen Shot 2019-07-05 at 3 46 19 PM](https://user-images.githubusercontent.com/5334504/60706480-2343a280-9f3c-11e9-8bb3-2230b7baa810.png)

Reply:
You mean sending a comma and one emoji like this?
![Screen Shot 2019-07-05 at 12 29 20 PM](https://user-images.githubusercontent.com/10777374/60742047-87586d80-9f20-11e9-8bcc-6a1c3c9cd62c.png)

Reply:
@a-arias he means the way you added the emojis (buy just `+`'ing the array) will add comma's between each item. If you wanted to add each emoji you'd have to  join the array.

Reply:
I understand now, I just push a fix for this. thanks for the clarification.

Reply:
So sorry for my confusing comment.  I liked you original approach but I'm also good with the current change.